staging/src/k8s.io/apiserver/pkg/storage/interfaces.go

```diff
@@ -73,16 +73,15 @@ type ResponseMeta struct {
 	ResourceVersion uint64
 }
 
-// MatchValue defines a pair (<index name>, <value for that index>).
-type MatchValue struct {
-	IndexName string
-	Value     string
-}
-
-// TriggerPublisherFunc is a function that takes an object, and returns a list of pairs
-// (<index name>, <index value for the given object>) for all indexes known
-// to that function.
-type TriggerPublisherFunc func(obj runtime.Object) []MatchValue
+// TriggerPublisherFunc is a function that for a given object computes
+// <value of an index> for a particular <index>.
+// TODO(wojtek-t): Rename to IndexerFunc?
```

Comment:
I can do this renaming if @wojtek-t hasn't started yet.

Reply:
If you have capacity, please do.

Reply:
See #80300